.github/issue_template.md

```diff
@@ -0,0 +1,27 @@
+### If it's a discussion, please start one [here](https://github.com/cultureamp/kaizen-design-system/discussions).
+
+- [ ] Before opening this issue, I've searched open and closed issues for a fix.
+
+## Expected Behavior
+
+
+## Actual Behavior (please provide screenshots)
+
+
+## Steps to Reproduce the Problem
+
+1.
+2.
+3.
+
+## Specifications (please add any other notable libs/details)
+
+- Kaizen version:
```

Comment:
I like including this, should go a long way to identifying if an issue might have been subtly fixed by a recent release

Reply:
Totally. Was also wondering if it's worth adding like a `Kaizen component version:` if it's like `"@kaizen/brand": "^1.1.1",` but maybe that's enough?